### ISSUE TITLE
I still think the Download button could be clearer

### ISSUE THREAD

**Issue by briandk** [SayreLab/DEAR-FacultyWebsiteIssues#2]
@sammyatu @smartinuk: To my eye, this download button still doesn't visually communicate that it's the primary action on the page.

![periscope__best-practices_classrooms](https://cloud.githubusercontent.com/assets/330036/6362992/1fb41f92-bc50-11e4-8602-f2f62fbe2dce.png)

Other than the fact that it's green, it looks like just about every other content header on the page (including tabs).

## My Suggestion

Consider adding something like a [download glyphicon][1] in the "Download" button that indicates what the button is for. Here's a terrible mock-up:

![periscope__best-practices_classrooms](https://cloud.githubusercontent.com/assets/330036/6363036/8d390ac8-bc50-11e4-89ac-e9f23b2eed20.png)

## Background

@sammyatu writes:

> FYI, Matt has now moved the download buttons over to where they're supposed to be on the Periscope pages. You can see what they look like on the live site here:
> 
> Lesson set:
> https://www.physport.org/periscope/sets/bp.cfm
> 
> Lesson:
> https://www.physport.org/periscope/lessons/bp/bp02.cfm

[1]: http://getbootstrap.com/components/#glyphicons

**Comment by sammyatu** [comment]
I'm copying in the whole original thread that sparked this discussion, which includes more issues related to this:

A couple things came up this week that have made me think about the download button for assessments, Periscope lessons, and other things on PhysPort, and I have a suggestion I'd like your thoughts on.

1) Sandy pointed out during my Periscope presentation that the Periscope download button didn't look like a button and suggested changing it from:
![image](https://cloud.githubusercontent.com/assets/11077995/6365338/ef728286-bc6a-11e4-8003-58d91266e901.png)
to:
![image](https://cloud.githubusercontent.com/assets/11077995/6365342/06ef2522-bc6b-11e4-9d02-4f76220a8dc1.png)
with the phrase "Download contains:" before the bullets.

2) Brian tested downloading assessments and I got to watch his reactions using this system for the first time. He was really disoriented when, after he was logged in and went back to the page he started at, the green arrow:
![image](https://cloud.githubusercontent.com/assets/11077995/6365346/14dce1c4-bc6b-11e4-9654-ab3807b54c09.png)
had changed to a green download button:
![image](https://cloud.githubusercontent.com/assets/11077995/6365350/2d7a4f32-bc6b-11e4-9881-471e30d28e6f.png)
that was smaller and no longer had text clearly telling him what it did.

3) My thought is that the download buttons should look the same for both assessments and Periscope lessons and lesson sets, and for any other packages that people download from the site, in order to create a consistent experience and expectation throughout. Also, that it might be helpful to have some text to communicate that the reason you are seeing this download button at all is because you are logged in as a verified educator. The two use cases this will address are: (1) user is confused and disoriented about why what they are seeing is not the same as before and (2) user is concerned about test security and doesn't remember that they logged in (because their computer did it automatically), so they think we are just letting anyone download these tests.

So my suggestion is two make the download button look like the assessment download button, but bigger so it's the same size as the arrow, for both Periscope and assessments, with text underneath it that says, "Download (you are a verified educator)" followed by a list of what the download contains (for assessments, this list would always be: cover sheet, assessment, answer key. I'm not exactly sure how to best lay this out on the page. I'm also not sure where, if at all, the version number and/or file size should go.

Thoughts?

**Comment by sammyatu** [comment]
Also, now that the Periscope download button is in there with real content, it is clear that Gabe did such a good job of cutting text on the lesson set descriptions that it looks really weird to have the whole list of contents under the download button because the download button contents take up way more vertical space than the lesson set description:
https://www.physport.org/periscope/sets/bp.cfm

**Comment by smartinuk** [comment]
OK, I see a few issues here:

1.  Brian found the change in icon on the assessment page from a right arrow to a download arrow confusing
2.  The download button on Periscope looks different from the download button everywhere else
3.  The list of stuff under the download button on periscope is too big, and it looks awkward.
4.  Sam suggested we should have some way of letting people know that they're logged in, so they don't think we're just letting anybody download assessments willy nilly

Please let me know if I missed anything.

Here are my thoughts on these items, in order.
1.  I think this change in icon is fine.  The right arrow has the word "login" underneath it.  The icon matches the action, and it appropriately sets the expectation that the user is going to have to *do something* before they get the assessment
2.  Yes, the more I think about it the more I agree with Sam that we could just use the same icon that we use for the assessments.  I think the text under the icon should just be " download zip (64k)" in tiny font, and we could have a hover tip that shows you the details of what's in the zip.  This means that we *wouldn't* be explicitly listing the contents of the lesson on the page, but I"m not sure how much that matters
3.  The idea for 2 solves issue 3
4.  The standard way to let people know they're logged in is with the account menu, typically found in the upper right.  It seems that PhysPort isn't really doing much in that regard at the moment... it just changes the word "login" to "logout".  If we want to be signifying peoples' logged-in state more clearly, I think this is the place to start (not on other places on the page)

Re-reading above, it seems that maybe there's some confusion about what things should look like for assessments (which have versions) vs Persicope lessons (which don't).  Do we need to engage with this?

**Comment by sammyatu** [comment]
Re 1: I think the issue is not so much that the icon changes - it *should* change because what you're allowed to do has changed - but that it doesn't clearly communicate what has changed and why. I think the text under the green arrow is super clear and really helpful. I propose that we make the download arrow more parallel to the green arrow by making it bigger (right now it is inexplicably smaller than the arrow), and replacing the version text with text in the same font as under the arrow that says "Download" in green and then "You are a verified educator." in gray.
Re 2: I am on board with changing the download button to match the assessments download button. I would modify the text slightly to say "Download lesson" or "Download lesson set" in the same green font we use on the assessment page, and then "64K zip file" in much smaller gray font. I do feel like we were trying to solve a problem by listing the contents - we were trying to communicate to people what kinds of things go into a lesson - so I'm reluctant to just not list the contents. One compromise would be not list the contents only on the lesson page, and not on the lesson set page, because the lesson page is where we want to do that education, and it has more vertical space because the lesson descriptions are longer than the lesson set description.
Re 3: I think my suggestion above addresses it.
Re 4: I think you've misunderstood my concern here. I don't think people will have any trouble finding that they're logged in, the problem is that I don't think they'll make the connection between that and why the icon has changed, or realize that they're only seeing the download option *because* they're logged in.

Re versions: No I don't think we need to engage in this at all. I don't think anyone cares what the version number is and we can safely just delete the version number from the download button. Anyone who does care can easily find the versions tab.

**Comment by smartinuk** [comment]
Sigh.  Many of these issues (size of icons, size of text, color of text) are visual design issues.  Do we have a visual designer anywhere?

**Comment by sammyatu** [comment]
No, we just need to do the best we can. You design a website with the
resources you've got, not the resources you wish you had. :)

On Wed, Feb 25, 2015 at 4:06 PM, Sandy Martinuk <notifications@github.com>
wrote:

> Sigh. Many of these issues (size of icons, size of text, color of text)
> are visual design issues. Do we have a visual designer anywhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SayreLab/DEAR-FacultyWebsiteIssues/issues/2#issuecomment-76090779>
> .
>



-- 
Sarah "Sam" McKagan

Director, PhysPort: www.physport.org
American Association of Physics Teachers

Physics Education Research consultant
McKagan Enterprises

**Comment by sammyatu** [comment]
OK, Sandy and I talked about this and came to the following conclusions:

Yes, make assessment download button bigger and put the word "download" in green underneath it.
No, don't put "you are a verified educator" underneath it.

Yes, change Periscope download button to match assessment download button.
Yes, put "Download lesson" or "Download lesson set" underneath it in the same green font we use on the assessment page, and then "64K zip file" in much smaller gray font.
Also, underneath that, also in small font, put "What's in this?" in font that is clearly a link, which opens up a pop-up window that lists the contents.

Sandy made me articulate the use cases for putting "you are a verified educator" under the download button. I think there are two: (1) people are confused and worried and want to know why they are allowed to download the assessment, and (2) we want to remind people of the benefits they are receiving as a result of having signed up as a verified educators in order to make them feel good about having done that and feel like they are receiving value. Sandy found (2) much more compelling, but felt that this was not the right place to address it. I don't think I agree, but I do think it makes sense to wait till usability testing to see if there is even a problem we need to solve before adding more text.

In the end, we don't know the right way to do this, so our goal is not to come up with the best design, but to come up with the design that will give us the most useful information in usability testing.
